### PR TITLE
Update allow-members-to-send-as-or-send-on-behalf-of-group.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated steps to reflect the new Exchange admin center.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7852

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7790 is also covered

File: microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md [hunk 1]
```diff
@@ -43,35 +43,32 @@ This section explains how to allow users to send email as a group in the<a href=
   
 1. In the Exchange admin center, go to **Recipients** \> <a href="https://go.microsoft.com/fwlink/?linkid=2183233" target="_blank">**Groups**</a>.
     
-2. Select **Edit**  ![Edit group icon.](../media/0cfcb590-dc51-4b4f-9276-bb2ce300d87e.png) on the group that you want to allow users to send as. 
+2. Select the Group that you want to allow users to send as.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md]:
```suggestion
2. Select the group that you want to allow users to send as. 
```

File: microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md [hunk 2]
```diff
@@ -43,35 +43,32 @@ This section explains how to allow users to send email as a group in the<a href=
   
 1. In the Exchange admin center, go to **Recipients** \> <a href="https://go.microsoft.com/fwlink/?linkid=2183233" target="_blank">**Groups**</a>.
     
-2. Select **Edit**  ![Edit group icon.](../media/0cfcb590-dc51-4b4f-9276-bb2ce300d87e.png) on the group that you want to allow users to send as. 
+2. Select the Group that you want to allow users to send as. 
     
-3. Select **group delegation**.
+3. Select **Settings** and Click on **Edit manage delegates**
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md]:
```suggestion
3. Select **Settings** > **Edit manage delegates**.
```

File: microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md [hunk 3]
```diff
@@ -43,35 +43,32 @@ This section explains how to allow users to send email as a group in the<a href=
...
     
-4. In the **Send As** section, select the **+** sign to add the users that you want to send as the Group. 
-    
-    ![Screenshot of send as dialog box.](../media/1df167f6-1eff-4f98-9ecd-4230fab46557.png)
+4. In the **Add a delegate** section, Type the email address of the user that you would like have send as acess.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md]:
```suggestion
4. In the **Add a delegate** section, enter the email address of the user that you would like to have **Send as** access.
```

File: microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md [hunk 4]
```diff
@@ -43,35 +43,32 @@ This section explains how to allow users to send email as a group in the<a href=
...
   
-5. Type to search or pick a user from the list. Select **OK** and **Save**.
+5. Select **Permission Type** as **Send as** from the drop down.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md]:
```suggestion
5. Select **Permission Type** as **Send as** from the drop-down.
```

File: microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md [hunk 5]
```diff
@@ -43,35 +43,32 @@ This section explains how to allow users to send email as a group in the<a href=
...
+
+6.  Click on **Save changes**
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md]:
```suggestion
6.  Select **Save changes**.
```

File: microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md [hunk 6]
```diff
@@ -43,35 +43,32 @@ This section explains how to allow users to send email as a group in the<a href=
...
-    ![Type to search or pick a user from the list.](../media/522919cf-664c-4a25-8076-c51c8c9fbe43.png)
-  
 ## Allow members to send email on behalf of a group
 
 This section explains how to allow users to send email on behalf of a group in the <a href="https://go.microsoft.com/fwlink/p/?linkid=2059104" target="_blank">Exchange admin center (EAC)</a> in Exchange Online.
   
 1. In the Exchange admin center, go to **Recipients** \> <a href="https://go.microsoft.com/fwlink/?linkid=2183233" target="_blank">**Groups**</a>.
     
-2. Select **Edit** ![Edit group icon.](../media/0cfcb590-dc51-4b4f-9276-bb2ce300d87e.png) on the group that you want to allow users to send as. 
+2. Select the Group that you want to allow users to send on behalf.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md]:
```suggestion
2. Select the group that you want to allow users to send on behalf of. 
```

File: microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md [hunk 7]
```diff
@@ -43,35 +43,32 @@ This section explains how to allow users to send email as a group in the<a href=
...
 
 This section explains how to allow users to send email on behalf of a group in the <a href="https://go.microsoft.com/fwlink/p/?linkid=2059104" target="_blank">Exchange admin center (EAC)</a> in Exchange Online.
   
 1. In the Exchange admin center, go to **Recipients** \> <a href="https://go.microsoft.com/fwlink/?linkid=2183233" target="_blank">**Groups**</a>.
     
-2. Select **Edit** ![Edit group icon.](../media/0cfcb590-dc51-4b4f-9276-bb2ce300d87e.png) on the group that you want to allow users to send as. 
+2. Select the Group that you want to allow users to send on behalf. 
     
-3. Select **group delegation**.
+3. Select **Settings** and Click on **Edit manage delegates**
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md]:
```suggestion
3. Select **Settings** > **Edit manage delegates**.
```

File: microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md [hunk 8]
```diff
@@ -43,35 +43,32 @@ This section explains how to allow users to send email as a group in the<a href=
...
     
-4. In the Send on Behalf section, select the **+** sign to add the users that you want to send as the Group. 
-    
-    ![Screenshot of send on behalf of dialog.](../media/2bae0579-8907-4d6b-8920-ddd6555897b4.png)
+4. In the **Add a delegate** section, Type the email address of the user that you would like have send as acess.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md]:
```suggestion
4. In the **Add a delegate** section, enter the email address of the user that you would like to have **Send as** access.
```

File: microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md [hunk 9]
```diff
@@ -43,35 +43,32 @@ This section explains how to allow users to send email as a group in the<a href=
...
   
-5. Type to search or pick a user from the list. Select **OK** and **Save**.
-    
-    ![Type to search or pick a user from the list.](../media/522919cf-664c-4a25-8076-c51c8c9fbe43.png)
+5. Select **Permission Type** as **Send on behalf** from the drop down.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md]:
```suggestion
5. Select **Permission Type** as **Send on behalf** from the drop-down.
```

File: microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md [hunk 10]
```diff
@@ -43,35 +43,32 @@ This section explains how to allow users to send email as a group in the<a href=
...
+
+6.  Click on **Save changes**
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md]:
```suggestion
6.  Select **Save changes**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**